independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/reader/QuteCodestartFileReader.java

```diff
@@ -50,6 +50,8 @@ public static String readQuteFile(CodestartResource projectResource, Source sour
         final Engine engine = Engine.builder().addDefaults()
                 .addResultMapper(new MissingValueMapper())
                 .removeStandaloneLines(true)
+                // TODO This needs to be revisited
+                .strictRendering(false)
```

Comment:
FYI @ia3andy

Reply:
Thanks!

Reply:
so what cases is it that you see that will break here ?

Reply:
I did not examine the failures but a lot of codestarts and devtools tests failed so it's definitely related. A quick scan example: _"Property "parent" not found on base object "java.util.HashMap" in expression {parent} in template 1 on line 6"_ in the`QuarkusExtensionCodestartGenerationTest`.

Reply:
interesting - now curious which of these are actual bugs.

should map lookups be considered valid or is expectation you do {obj.parent ? false} ?

sucks we didnt notice this before 2.0 :/

Reply:
I'll need to take a look at the tests. But it can be a bug in the test too ;-).

Reply:
So I checked the tests and the problematic pattern looks like `{#if uberjar}<quarkus.package.type>uber-jar</quarkus.package.type>{/if}`. Previously this didn't fail because "not found" was considered a `falsy` value. In this PR, we assume that "not found" cannot be evaluated correctly by default and thus the rendering fails. Note that `null` would be OK because it means that we found something...

With strict rendering enabled the template would have to use something like `{#if uberjar.or(false)}` (elvis operator would not work here because it's a param of a section) or even `{#if containsKey('uberjar')}` (because the data object is a manually created `Map`).

That said, I think it's ok to disable the strict rendering for situations like these. I'll let @ia3andy to decide later. For now, I will add a more explaining comment.

> should map lookups be considered valid or is expectation you do {obj.parent ? false} ?

I don't think we should add a special handling for any type. That might be even more confusing.